storage/innobase/buf/buf0flu.cc

```diff
@@ -63,7 +63,7 @@ static const ulint buf_flush_wait_flushed_sleep_time = 10000;
 #include <my_service_manager.h>
 
 /** Number of pages flushed through non flush_list flushes. */
-static ulint buf_lru_flush_page_count = 0;
+ulint buf_lru_flush_page_count = 0;
```

Comment:
I would suggest declaring this variable `ulong` or `ulonglong` so that it can be bound to the `innodb_status_variables` directly.

Please omit the explicit initialization. Global or `static` variables are guaranteed to be zero-initialized in the BSS. Older compilers could generate redundant code for zero-initializing the variable.